packages/angular/projects/clr-angular/src/forms/common/form.ts

```diff
@@ -20,8 +20,9 @@ import { ClrLabel } from './label';
 })
 export class ClrForm {
   @Input('clrLabelSize')
-  set labelSize(size: number) {
-    this.layoutService.labelSize = size;
+  set labelSize(size: number | string) {
+    const sizeNumber = parseInt(size as string);
```

Comment:
We will need to cover the case when `size` is `''`.  `paseInt` will return `NaN`

Reply:
There's additional validation in `LayoutService`. If the value is `NaN`, it won't be set.

Reply:
```js
const sizeNumber = parseInt(size as string, 10) || 0;
```

* handle the `NaN` case by setting it to 0 when it could not find a number for it (if there is some default value of size - better use it then 0 .. it's just an example)
* setting `radix` will handle the conversion when it comes to passing strange forms of numbers ;) 

```js
parseInt("09", 8) 
// => 0
parseInt("09", 10)
// => 9
```
Not all browser will use default `10`

Reply:
Thanks! I've updated the PR.